.github/workflows/release.yml

```diff
@@ -47,40 +45,19 @@ jobs:
             echo "❌ One or both workflows failed — aborting."
             exit 1
           fi
+          
+      - name: Install tools
+        run: |
+          pip install --upgrade tomlkit
 
       - name: Checkout code
         uses: actions/checkout@v4
-
-      - name: Check for recent release commit
```

Comment:
so if I understand correctly, we don't need this anymore because we just trigger whenever we want